File: DSA/SakshamDwivedi_590013147/Day32/D32P1.py
#Solution Function
def swap(num1, num2):
    num1 = num1 ^ num2
    num2 = num1 ^ num2
    num1 = num1 ^ num2
    return num1, num2

#Fixed Input
print("Num 1 : ", swap(1,2)[0], "Num 2 : ", swap(1, 2)[1])

#Custom Input
num1 = int(input("Enter number 1 : "))
num2 = int(input("Enter number 2 : "))
print("Num 1 : ", swap(num1, num2)[0], "Num 2 : ", swap(num1, num2)[1])

File: DSA/SakshamDwivedi_590013147/Day32/D32P1.c
#include <stdio.h>

int num1;
int num2;

int main(){
  printf("Enter number 1 : ");
  scanf("%d", &num1);
  printf("Enter number 2 : ");
  scanf("%d", &num2);
  num1 = num1 ^ num2;
  num2 = num1 ^ num2;
  num1 = num1 ^ num2;
  printf("Number 1 : %d and Number 2 : %d\n", num1, num2);

  return 0;
}
